lib/msf/core/payload/windows/x64/reverse_http.rb

```diff
@@ -29,7 +29,8 @@ def initialize(*args)
     super
     register_advanced_options([
         OptInt.new('StagerURILength', [false, 'The URI length for the stager (at least 5 bytes)']),
-        OptInt.new('StagerRetryCount', [false, 'The number of times the stager should retry if the first connect fails', 10]),
+        OptInt.new('StagerRetryCount', [false, 'The number of times the stager should retry if the first connect fails (zero to infinite retries)', 10]),
+        OptInt.new('StagerRetryWait', [false, 'Number of seconds to wait for the stager between reconnect attempts']),
```

Comment:
Again, would prefer a sane default rather than supporting `nil`. I don't think anyone is interested in a stager that doesn't wait between retries.